tests/phpt/spool_drain.phpt

```diff
@@ -1,68 +0,0 @@
---TEST--
-Test emptying spool transport
---FILE--
-<?php
-
-declare(strict_types=1);
-
-namespace Sentry\Tests;
-
-use PHPUnit\Framework\Assert;
-use Sentry\ClientBuilder;
-use Sentry\Options;
-use Sentry\SentrySdk;
-use Sentry\Spool\MemorySpool;
-use Sentry\Spool\SpoolInterface;
-use Sentry\Transport\NullTransport;
-use Sentry\Transport\SpoolTransport;
-use Sentry\Transport\TransportFactoryInterface;
-use Sentry\Transport\TransportInterface;
-
-$vendor = __DIR__;
-
-while (!file_exists($vendor . '/vendor')) {
-    $vendor = dirname($vendor);
-}
-
-require $vendor . '/vendor/autoload.php';
-
-$spool = new MemorySpool();
-$nullTransport = new NullTransport();
-
-$transportFactory = new class($spool) implements TransportFactoryInterface {
-    private $spool;
-
-    public function __construct(SpoolInterface $spool)
-    {
-        $this->spool = $spool;
-    }
-
-    public function create(Options $options): TransportInterface
-    {
-        return new SpoolTransport($this->spool);
```

Comment:
Is this test not replaceable? Maybe we can extend/decorate the spool transport?

Reply:
I deleted this entire test because there was no real point in keeping it: both the transport and the spool are already tested elsewhere, and this PHPT test was just combining the two by asserting that everything worked into a shutdown function which in no way influence the final result